subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -1,220 +1,192 @@
-Users can choose to render the full graph of dependencies as well as identify the selection reason and origin for a dependency.
-The origin of a dependency can be a declared dependency in the build script or a transitive dependency in graph plus their corresponding configuration.
-Gradle offers both capabilities through visual representation via build scans and as command line tooling.
-
 [[sec:debugging-build-scans]]
-== Build scans
+= View and Debug Dependencies
 
-[NOTE]
-====
-If you do not know what link:https://scans.gradle.com/[build scan]s are, be sure to check them out!
-====
+Gradle provides tooling to navigate dependency graphs and mitigate link:https://en.wikipedia.org/wiki/Dependency_hell[dependency hell].
+Users can render the full graph of dependencies as well as identify the selection reason and origin for a dependency.
+Dependencies can originate through build script declared dependencies or transitive dependencies.
+You can visualize dependencies with the built-in Gradle CLI `dependencies` task or with link:https://scans.gradle.com/[build scans].
 
-A build scan can visualize dependencies as a navigable, searchable tree.
-Additional context information can be rendered by clicking on a specific dependency in the graph.
+[[sec:listing_dependencies]]
+== List Project Dependencies
 
-.Dependency tree in a build scan
-image::dependency-management-dependencies-report-build-scan.png[]
+Gradle provides the built-in `dependencies` task to render a dependency tree from the command line.
+By default, the dependency tree renders dependencies for all <<declaring_dependencies.adoc#sec:what-are-dependency-configurations,configurations>>, but only a <<command_line_interface#executing_tasks_in_multi_project_builds,single project>>.
+The dependency tree indicates the selected version of each dependency.
+It also displays information about dependency conflict resolution.
 
-[[sec:listing_dependencies]]
-== Listing dependencies in a project
+=== Output Annotations
 
-Gradle can visualize the whole dependency tree for every <<declaring_dependencies.adoc#sec:what-are-dependency-configurations,configuration>> available in the project.
+The `dependencies` task marks dependency trees with the following annotations:
 
-Rendering the dependency tree is particularly useful if you'd like to identify which dependencies have been resolved at runtime.
-It also provides you with information about any dependency conflict resolution that occurred in the process and clearly indicates the selected version.
-The dependency report always contains declared and transitive dependencies.
+- `(*)`: Repeat occurrence of a transitive dependency subtree. Gradle expands transitive dependency subtrees only once per project; repeat occurrences only display the root of the subtree, followed by this annotation.
+- `(c)`: Dependencies whose version is limited by dependency constraints.
```

Comment:
The annotation comment is wrong.

Here is the full explanation:

Indicates that the element is a dependency constraint. The representation never follows transitives from a constraint. Look for the matching dependency elsewhere in the tree.

Reply:
Brilliant, this makes much more sense to me. I'll also add this to the CLI output because the current description just says "dependency constraint" with no additional info.

Reply:
Is this a more accurate reword?
```suggestion
- `(c)`: A dependency constraint directly limits the version of this element. Does not apply to indirect constraints introduced transitively through another element's constraint.
```

Reply:
Fixed!